lms/templates/seq_module.html

```diff
@@ -19,8 +19,8 @@
            data-page-title="${item['page_title']|h}"
            aria-controls="seq_contents_${idx}"
            id="tab_${idx}"
-           tabindex="0"
-           role="tab">
+           tabindex="0">
+            <i class="icon fa seq_${item['type']}"></i>
```

Comment:
So, the "Type" is not accessible to a screen reader user.   We should be exposing SR text that is an equivalent of whatever that icon represents so the screen reader user knows if the unit is an assessment, a video, text, etc.  I'm crossing my fingers that the 'type' in {item['type']} is a functional equivalent for describing the unit, otherwise we will have to create a mapping to human readable equivalents.